spec/cloud.go

```diff
@@ -153,6 +154,30 @@ func (g *EC2Generator) Generate() (interface{}, error) {
 	return results, nil
 }
 
+// SuggestCustomIdentifier suggests the identifier of the EC2 instance
+func (g *EC2Generator) SuggestCustomIdentifier() (*string, error) {
```

Comment:
it's not so much complicated code, but adding some simple test is preferable, i think